scipy/stats/sampling.py

```diff
@@ -66,3 +66,8 @@
    SimpleRatioUniforms,
    UNURANError
 )
+
+__all__ = ["NumericalInverseHermite", "NumericalInversePolynomial",
+           "TransformedDensityRejection", "SimpleRatioUniforms",
+           "RatioUniforms", "DiscreteAliasUrn", "DiscreteGuideTable",
+           "UNURANError", "FastGeneratorInversion"]
```

Comment:
@chrisb83 git blame points at you: any reason `scipy.stats.sampling` did not have an `__all__` list, and would you object if it gets one?

Reply:
I think I followed `stats/qmc.py`. I don't see a problem with the addition of __all__